src/scripts/scenes/TileVisual.ts

```diff
@@ -298,6 +319,20 @@ export class TileVisual extends Container {
                this._level.text = getBuildingLevelLabel(this._tile.building);
             }
 
+            const options = getGameOptions();
+            if (options.mapStoragePercentage === true) {
+               const storage = getStorageFor(this._xy, this._gs);
```

Comment:
`getStorageFor` is not a particularly cheap function. I am not sure how this would impact performance. Ideally that function needs to be optimized before adding it here.